spec/unit/serializer_spec.rb

```diff
@@ -4,6 +4,7 @@
 require 'iknow_params'
 
 require 'rails'
+require 'active_support/core_ext'
```

Comment:
Why is this now required?

Reply:
I'm not sure, but new activesupport failed without it.